src/components/calcite-tab-nav/calcite-tab-nav.tsx

```diff
@@ -116,12 +132,55 @@ export class CalciteTabNav {
     }
   }
 
+  render(): VNode {
+    this.dir = getElementDir(this.el);
+
+    const indicatorStyle =
+      this.dir !== "rtl"
+        ? {
+            width: `${this.indicatorWidth}px`,
+            left: `${this.indicatorOffset}px`
+          }
+        : {
+            width: `${this.indicatorWidth}px`,
+            right: `${this.indicatorOffset}px`
+          };
+    return (
+      <Host role="tablist">
+        <div
+          class="tab-nav"
+          onScroll={() => this.handleContainerScroll()}
```

Comment:
nitpick: I think you could do `private handleContainerScroll = (): void => {` and then just `onScroll={this.handleContainerScroll}`